validator/sawtooth_validator/state/client_handlers.py

```diff
@@ -25,189 +25,253 @@
 
 from sawtooth_validator.protobuf import client_pb2
 from sawtooth_validator.protobuf.block_pb2 import BlockHeader
-from sawtooth_validator.protobuf.state_context_pb2 import Entry
 from sawtooth_validator.protobuf import validator_pb2
 
 
 LOGGER = logging.getLogger(__name__)
 
 
-class StateCurrentRequestHandler(Handler):
+class StateCurrentRequest(Handler):
```

Comment:
These could probably use some doc string, but could be handled as a separate commit (since they weren't introduced here).